tests/bal_functional_behaviour_suite.py

```diff
@@ -0,0 +1,169 @@
+#
+#   Copyright 2013-2023 The Foundry Visionmongers Ltd
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+"""
+A manager test harness test case suite that validates that the
+BasicAssetLibrary manager behaves as expected concerning functional
+behaviors that don't fall under "business-logic", such as performance
+related requirements.
+"""
+
+# pylint: disable=invalid-name, missing-function-docstring, missing-class-docstring
+
+import os
+from unittest import mock
+
+from openassetio import Context, TraitsData
+from openassetio.test.manager.harness import FixtureAugmentedTestCase
+
+
+class Test_simulated_latency_default(FixtureAugmentedTestCase):
+    shareManager = False
+    """
+    Test that runs in isolation to check the default initialization
+    behaviour for the latency setting.
+    """
+
+    def test_when_latency_not_set_default_value_is_used(self):
```

Comment:
[consider] As this leaves a lot to remembering how the test harness works
```suggestion
    def test_when_uninitialized_then_defaults_to_10ms(self):
```

Reply:
d89dc1361f179c43023e141264ecf22d79d14292